src/chrome/content/rules/UPS.com.xml

```diff
@@ -22,28 +22,42 @@
 <ruleset name="UPS.com (partial)">
 
 	<target host="ups.com" />
-	<target host="*.ups.com" />
+		<exclusion pattern="^http://ups.com/$" />
+	<target host="www.ups.com" />
+		<exclusion pattern="^http://www.ups.com/$" />
+	<target host="wwwapps.ups.com" />
+	<target host="www.apps.ups.com" />
+	<target host="campusship.ups.com" />
+	<target host="www.campusship.ups.com" />
+
 		<!--
 			Redirects to http:
 						-->
-		<!--exclusion pattern="^http://www\.ups\.com/content/globalentry$" /-->
-		<!--
-			all ur pkgs r now belong to nsa
+		<exclusion pattern="^http://www\.ups\.com/content/globalentry$" />
+			<test url="http://www.ups.com/content/globalentry" />
 
-			(Now redirects to login:
-						-->
-		<!--exclusion pattern="^http://www\.ups\.com/WebTracking/track($|\?)" /-->
 		<!--
-			Exceptions:
-					-->
-		<exclusion pattern="^http://www\.ups\.com/+(?!assets/|favicon\.ico|img/|styles/|stylesheets/)" />
-
+			Now redirects to login:
+						-->
+		<exclusion pattern="^http://www\.ups\.com/WebTracking/track($|\?)" />
+			<test url="http://www.ups.com/WebTracking/track" />
+			<test url="http://www.ups.com/WebTracking/track?" />
```

Comment:
Ideally for these types of test URLs we should have query parameters that are meaningful. The goal is not just to satisfy the letter of the coverage rule, but to let the coverage rule be a reminder that the more complex the rewrite rule is, the more likely it is that one of the types of pages it covers will break, so we want to test all the different types of pages.

It may not be possible to get a useful tracking URL that does not expose private data, though. In this case I'd be willing to merge non-valid test URLs for that reason.